speaking-guidelines.md

```diff
@@ -0,0 +1,70 @@
+
+Thank you for your interest in presenting to Triangle DevOps. We very much
+appreciate it.
+
+We have a few guidelines we ask you to understand about presenting to our group.
+
+## Preparing your talk
+
+First, talks should aim for about 45 minutes to an hour of length. Lean for
+shorter than an hour if you're expecting your talk to generate questions.
+
+Please let us know how long you're expecting your talk to run ahead of time so
+we can make sure time is budgeted accordingly.
+
+Second, we have a mix of experiences and backgrounds in our group. Be careful
+about assuming that the acronyms, terms, concepts and skills you'll be speaking
+about are universally understood. We want to hear from a broad range of opinion,
+experience and expertise. Help bring our members along with you by taking a
+moment to explain acronyms and briefly explain foundational concepts. Look over
+your material and ask what a newcomer to the field might not understand.
+
+Third, please review our [Code of Conduct][conduct] with respect to your talk.
+We want your talk to reflect the inclusiveness we seek for our community. To
+that end, we kindly ask as you prepare that you:
+
+* Use non-gendered, inclusive, language like "folks" over "guys"
+* Be mindful that animated GIFs, memes and memespeak may have a limited
+  audience. They're fine to use within the boundaries of the Code of Conduct,
+  but make sure you read the room to see if everyone's catching on. When in
+  doubt, explain the meaning behind the GIF, meme or memespeak.
+
+If you have questions about your talk, whether it's concepts or specific
+materials, we will be happy to discuss either with you.
+
+Technically speaking, we can generally accommodate VGA and Mini DisplayPort
+connections at the venues we use. If you need a different video connection,
+please let us know so we can coordinate and make arrangements ahead of time.
+
+[conduct]: code-of-conduct.md
+
+## Before your talk
+
+Our monthly meetups, unless otherwise announced, start at 7. Doors may open
+earlier, at the venue's option. 6:45 PM is typical. The first 10 to 15
+minutes from 7 PM are for networking for the attendees. Please arrive before or
+as close to 7 PM as you can. Let the venue host or an organizer know when you've
+arrived, and we'll show you where you'll be presenting.
+
+We'll begin the program with announcements, a call for "who's hiring", and then
+introduce you. You're on!
+
+## During your talk
+
+* We generally don't have microphones, so please project
+* If you are asked questions from the audience, please repeat the question. It's
+  typically easier to hear you in the audience than it is an audience member on
+  the other side of the room
+* If it seems like your talk is running long, we will attempt to accommodate
+  with the venue, but we will not always be able to to do. Generally, if you're
+  running longer than an hour of speaking time, we'll give you a five minute
+  warning to bring your talk to a conclusion
```

Comment:
I typically aim to hard stop at 8:30 if we've gone that long - that's often enough time for the socializing + announcements and then the hour talk.

If the venue permits, we'll often allow the community more social time and a chance to interact 1 on 1 with the speaker after the talk for a bit.